frontend/src/scenes/cohorts/CohortFilters/CohortField.scss

```diff
@@ -19,18 +19,40 @@
 
 .CohortField__CohortPersonPropertiesValuesField {
     min-width: 150px;
-    height: 40px;
+    min-height: 40px;
     border: 1px solid var(--border);
     border-radius: var(--radius);
 
-    .ant-select-selector {
-        height: 100% !important;
-        border: none !important;
+    .ant-select-selection-search,
+    .ant-select-selection-placeholder {
+        display: flex;
+        align-items: center;
+        padding: 0 4px !important;
+    }
+
+    &.ant-select-single {
+        .ant-select-selector {
+            min-height: 38px !important;
+            height: unset;
+            border: none !important;
+            background-color: inherit;
+        }
+    }
+
+    &.ant-select-multiple {
+        .ant-select-selector {
+            height: 100% !important;
+            border: none !important;
+            padding: 5px 40px 5px 11px;
+            background-color: inherit;
+
+            .ant-select-selection-search {
+                padding-left: 0px !important;
+            }
 
-        .ant-select-selection-search,
-        .ant-select-selection-placeholder {
-            display: flex;
-            align-items: center;
+            .ant-select-selection-placeholder {
+                padding-left: 6px !important;
+            }
```

Comment:
ugly but necessary if we want to continue using antd. Decided against using LemonSelectMultiple here since that would be a much bigger refactor.